src/logger.py

```diff
@@ -27,3 +28,21 @@ def __init__(self, config_path, name='root'):
     def log_message(self, log_level, msg):
```

Comment:
Since we have separate methods for all log levels now and the `log_message` method is only being accessed within the class, how about declaring it `protected`?
```suggestion
    def _log_message(self, log_level, msg):
```

Reply:
That's not necessary, we need to get changed merged quicker and focus on the real issues.  Feel free to send a PR with follow-up improvements if you want, I think we could just merge Logger into Service.

Reply:
Well, In my opinion, this change is more relevant to this PR as logging methods specific to log levels have been introduced here.
Hence, it is more suitable if we can have the suggested change along with this PR itself.
Otherwise, feel free to merge the PR.

Reply:
It's still perfectly fine to do `Logger.log_message()` even if we have other methods that make the syntax simpler.